packages/builtin_test/test.pony

```diff
@@ -877,6 +878,63 @@ class iso _TestDivMod is UnitTest
     h.assert_eq[I64](-5, -13 % -8)
 
 
+class iso _TestAddc is UnitTest
+  """
+  Test addc on various bit widths.
+  """
+  fun name(): String => "builtin/Addc"
+
+  fun apply(h: TestHelper) =>
+    test_addc[U8](h, (0xff, false), U8(0xfe).addc(   1))
+    test_addc[U8](h, (   0, true),  U8(0xff).addc(   1))
```

Comment:
Thanks for doing this PR, but can you remove the extra alignment whitespace here?  I personally like alignment whitespace in many cases, but it's not consistent with the existing style in the Pony codebase.